recipes-extended/xen/xen-common.inc

```diff
@@ -72,6 +72,7 @@ SRC_URI_append = " \
     file://libxl-add-ovmf-varstored.patch \
     file://libxl-const-stubdom-is-linux.patch \
     file://libxl-disable-dom0-qemu.patch \
+    file://libxl-xen-disable-dom0-qemu.patch \
```

Comment:
Not necessarily an issue, but in the interest of keeping the patch queue trimmed, do we want to combine these two qemu patches? They provide the same fix essentially.

Reply:
The two patches come from Qubes - I've wondered why they are separate there.  Since they come from elsewhere, I think we should just leave them as-is.

Reply:
Ack